Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -421,6 +421,15 @@ public void UnsetMessageCountSemaphore()
             m_messageCountSemaphoreName = null;
         }
 
+        /// <summary>
+        /// Optional AnyBuild shim execution information.
+        /// </summary>
+        /// <remarks>
+        /// Internal since this is set as a side effect of initializing a FileAccessManifest
+        /// in <see cref="SandboxedProcessInfo"/>.
+        /// </remarks>
+        internal AnyBuildShimInfo AnyBuildShimInfo { get; set; }

```

Comment:
`[CanBeNull]` attribute untill we use the C#'s [https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/) feature